README.md

```diff
@@ -70,4 +70,4 @@ Give a ⭐️ if you like this project!
 
 ## 📝 License
 
-This project is [MIT](https://github.com/git/git-scm.com/blob/main/MIT-LICENSE.txt) licensed.
+This project is [MIT](LICENSE.md) licensed.
```

Comment:
- [ ] Great job adding the README but I think you forget to deoply the project since I can't see the link in the README file so please deploy it and add it since it is of one the requirements 🚀🔥😊💯


- [ ] I think it would be great if you add in the acknowledgemnt the original design belonges to who so kindly add it

![image](https://user-images.githubusercontent.com/58553711/192107725-cfdbc5a4-2f2f-4ae1-90be-3fc5e5eda0ce.png)

Reply:
Thank you!